docs/prosessere-data/notebooks.md

```diff
@@ -133,48 +145,15 @@ Begge deler kan gjøres via [Vault CLI](https://www.vaultproject.io/docs/command
 
 Når du har hentet ut hemmeligheten fra Vault må du legge det inn i Google Secret manager som beskrevet [ovenfor](#credentials-og-hemmeligheter).
 
-## Bruk av hostnavn på onprem-tjenester
-I dette prosjektet støtter vi ikke navneoppslag mot on-premises.
-Det betyr at databasenavn må oversettes til ip-adresser i Notebooks.
-Man kan finner ip-adressen til en database ved å pinge hostnavnet fra f.eks. utviklerimage:
-```
-$ ping dm08db01.adeo.no
-> PING dm08db01.adeo.no (10.x.x.x): 56 data bytes
-```
-
-Vi har også lagd en liste over de åpningen vi har (og ip-adressene) over i [navikt/pig/nada](https://github.com/navikt/pig/blob/master/nada/doc/knada-gcp.md#brannmur%C3%A5pninger).
-
-For å bruke IP i stedet for en vanlig adresse så er det bare å bytte direkte.
-Så hvis følgende er den "vanlige" kode-snutten din:
-```python
-dsnStr = cx_Oracle.makedsn('dm08db01.adeo.no','1521',service_name='DWH_HA')
-```
-
-så kan du heller skrive
-```python
-dsnStr = cx_Oracle.makedsn('10.x.x.x','1521',service_name='DWH_HA')
-```
-
-## Maskintype og GPU
-Det er du som vet best hva du trenger, derfor er det ingen begrensninger på hva du kan velge av maskin og GPU.
-Bare husk at det kan bli veldig kostbart hvis du lar en maskin (med mye minne og GPU) stå uten at den blir brukt.
-
-## Stoppe en maskin
-Har du en maskin du ikke trenger for en periode, men ikke ønsker å [slette](#slette-en-maskin), så kan du stoppe den.
-Velg maskinen i [oversikten](https://console.cloud.google.com/ai-platform/notebooks/instances) og trykk `STOP` i toppen.
-
-## Slette en maskin
-Har du en maskin du ikke trenger lenger, så kan du slette den.
-Velg maskinen i [oversikten](https://console.cloud.google.com/ai-platform/notebooks/instances) og trykk `DELETE` i toppen.
-
 ## Oppsett for Oracle og Postgres drivere
-For å gjøre det enkelt for dere å komme i gang, har vi lagd to scripts som begge må kjøres med root privilegier.
```

Comment:
Legge til en setning om hva drivere er/ hvorfor dette er nyttig å gjøre?

Reply:
Merker jeg er veldig usikker på hvilket nivå jeg skal legge meg på når det gjelder spørsmålet "hva drivere er". Kunne det holdt å lenke til https://en.wikipedia.org/wiki/Device_driver?
> hvorfor dette er nyttig å gjøre?
Lurer du på hvorfor det er nyttig å ha en driver?


Kan dette være nok?
> (en driver (i vår kontekst) er et program man må ha for å snakke med en database)

Reply:
Synes forslaget ditt er nok for å forklare hva det er 👍 . Tenker det kan være greit for de som ikke er kjent med begrepet at det forklares siden det står i tittelen, og at det kan forklare eller gjøre "hvorfor" tydeligere. Når jeg leste dokumentasjonen forsto jeg ikke hvorfor jeg burde gjøre dette/ hva skriptene gjør.